{"name": "Invalid operator", "status": "passed", "steps": [{"name": "When I enter \"5\" as the first number", "status": "passed", "start": 1689280813795, "stop": 1689280813805}, {"name": "And I enter \"@\" as the operation", "status": "passed", "start": 1689280813805, "stop": 1689280813806}, {"name": "And I enter \"3\" as the second number", "status": "passed", "start": 1689280813807, "stop": 1689280813807}, {"name": "Then the answer should be \"invalid operator\"", "status": "passed", "start": 1689280813807, "stop": 1689280813808}], "start": 1689280813792, "stop": 1689280813808, "uuid": "acedac37-74af-4893-ad2b-c0eb3d100a17", "historyId": "1351c9a1fd2ef7433d072e396fa399d3", "fullName": "Calculator logic test: Invalid operator", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Calculator logic test"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}